tmt/steps/provision/testcloud.py

```diff
@@ -981,7 +979,7 @@ def start(self) -> None:
             raise ProvisionError(f"Image '{self._image.local_path}' not found.") from error
         except (testcloud.exceptions.TestcloudPermissionsError, PermissionError) as error:
             raise ProvisionError(
-                f"Failed to prepare the image. Check the '{TESTCLOUD_IMAGES}' "
+                f"Failed to prepare the image. Check the '{self.config.STORE_DIR}' "
```

Comment:
`self.config.STORE_DIR` should become `self.testcloud_image_dirpath`.

Reply:
@skycastlelily this one looks unresolved?